src/utils/spp.ts

```diff
@@ -0,0 +1,55 @@
+import request, { CliRequestOptions } from '../request.js';
+
+export interface ContentCenterSite {
+  WebTemplateConfiguration: string;
+}
+
+export interface SppModel {
+  AIBuilderHybridModelType: string;
+  AzureCognitivePrebuiltModelName: string;
+  BaseContentTypeName: string;
+  ConfidenceScore: string;
+  ContentTypeGroup: string;
+  ContentTypeId: string;
+  ContentTypeName: string;
+  Created: string;
+  CreatedBy: string;
+  DriveId: string;
+  Explanations: string;
+  ID: number,
+  LastTrained: string;
+  ListID: string;
+  ModelSettings: string;
+  ModelType: number,
+  Modified: string;
+  ModifiedBy: string;
+  ObjectId: string;
+  PublicationType: number;
+  Schemas: string;
+  SourceSiteUrl: string;
+  SourceUrl: string;
+  SourceWebServerRelativeUrl: string;
+  UniqueId: string;
+}
+
+export const spp = {
+  /**
+   * Asserts whether the specified site is a content center
+   * @param siteUrl The URL of the site to check
+   */
+  async assertSiteIsContentCenter(siteUrl: string): Promise<void> {
+    const requestOptions: CliRequestOptions = {
+      url: `${siteUrl}/_api/web?$select=WebTemplateConfiguration`,
+      headers: {
+        accept: 'application/json;odata=nometadata'
+      },
+      responseType: 'json'
+    };
+
+    const response = await request.get<ContentCenterSite>(requestOptions);
```

Comment:
Since we only retrieve the `WebTemplateConfiguration` property, we don't really need this interface since all other properties will be `undefined`. Let's use `{ WebTemplateConfiguration: string }` as type.